runner/ansible/roles/2.2.3.exclude/tasks/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+
+- name: Set Test ID
```

Comment:
Can we run this checks with ansible native methods? https://docs.ansible.com/ansible/latest/collections/ansible/builtin/package_module.html

Reply:
yes, we should be able to do that/